include/BandLimitedWave.h

```diff
@@ -52,17 +52,17 @@ typedef struct
 public:
 	inline sample_t sampleAt(int table, int ph)
 	{
-		if (table % 2 == 0) {
-			return m_data[TLENS[table] + ph];
-		} else {
+		if (table % 2 == 0) { return m_data[TLENS[table] + ph]; }
```

Comment:
The ``if`` is on a single line, but the ``else`` is not. It looks inconsistent.

Reply:
Putting the else completely into one line [seems impossible](https://clang.llvm.org/docs/ClangFormatStyleOptions.html) (possibly a bug or limitation to `AllowShortIfStatementsOnASingleLine`). As soon as I turn this `else` into an `if`, it works.

I can only change it to

```
                if (table % 2 == 0) { return m_data[TLENS[table] + ph]; }
                else {
                        return m_data3[TLENS[table] + ph];
                }
```

Is that better or worse?

Reply:
This looks better:
``
                if (table % 2 == 0)
                { 
                        return m_data[TLENS[table] + ph]; 
                }
                else 
                {
                        return m_data3[TLENS[table] + ph];
                }
``

A ternary if is even better:
``
                return (table % 2 == 0) ? m_data[TLENS[table] + ph] : m_data3[TLENS[table] + ph];
``

Reply:
I know, but the coding rules said `If the block can fit on one line, it is acceptable`. I think we must live with the bad else.

Using a ternary is nice, but clang-format can't do that.